tests/unit/adapters/application-test.js

```diff
@@ -502,8 +698,10 @@ test('#fetch handles 4xx (Client Error) response status', function(assert) {
   });
   let promise = adapter.fetch('/posts', { method: 'POST', body: 'json string here' });
   assert.ok(typeof promise.then === 'function', 'returns a thenable');
-  promise.catch(function(error) {
+  promise.catch((error) => {
     assert.ok(error.name, 'Client Error', '4xx response throws a custom error');
+    // This one doesn't really test anything, does it? It checks if the mocked response
+    // above uses the correct format (errors array).
```

Comment:
Yeah not testing the actual code, more of "testing" that includes documentation as well.